scripts/extensions/ai-widget/src/requests.ts

```diff
@@ -0,0 +1,12 @@
+import {IArticle, ISuperdesk} from 'superdesk-api';
+
+export interface IRequests {
+    generateHeadlines?: (article: IArticle, superdesk: ISuperdesk) => Promise<Array<string>>;
```

Comment:
It should be mandatory. In `activate` method of the extension, check if it is set, and if not - display an error alert about wrong configuration and don't return anything from activate method so it doesn't get registered.